cloudinit/sources/DataSourceNoCloud.py

```diff
@@ -408,6 +436,8 @@ def ds_detect(self):
             if serial == "nocloud-net":
                 log_deprecated()
             return True
+        elif self.sys_cfg.get("datasource", {}).get("NoCloud").get("seedfrom"):
```

Comment:
Do you think it is worth {unit}testing this?

Reply:
Done